jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -4,11 +4,28 @@
     :link="docs_link || 'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#subset-tools'"
     :popout_button="popout_button">
 
+    <v-row>
+      <div style="width: calc(100% - 32px)">
+      </div>
+      <div style="width: 32px">
+        <j-tooltip tipid='plugin-plot-options-multiselect-toggle'>
```

Comment:
do we want a dedicated tooltip here that mentions that its only useful for recentering?

Reply:
I can certainly add that!

Reply:
So this means that multi-select should not even be possible unless you detect we are in Imviz, right? Because recentering is only enabled in Imviz.

Reply:
Good catch, I think that's right.  Although technically we probably could allow recentering in any image viewer (i.e. cubeviz), but _currently_ its restricted in the vue file to only be enabled for Imviz.